tests/dragonfly/snapshot_test.py

```diff
@@ -500,3 +502,33 @@ async def test_tiered_entries_throttle(async_client: aioredis.Redis):
 
     await load_task
     assert await StaticSeeder.capture(async_client) == start_capture
+
+
+@dfly_args({"proactor_threads": 1, "dbfilename": "big_snapshot"})
+async def test_big_value_serialization_memory_limit(df_factory):
+    instance = df_factory.create()
+    instance.start()
+    client = instance.client()
+
+    ten_mb = 125000
```

Comment:
This value is named oddly. ten_mb means 10 mega bytes? Because it has a value of 125k, so what is that about? :rofl:

Reply:
Ten megabits! Suggest a rename and I will do it!

Reply:
I still don't follow.. what is this number? 10 megabits is `10*2^20`, which is 10485760. It's also not the number of bytes in 10 megabits.
And also, why do we care about bits here? :rofl: